aws/lambdas/code/audit_logs_archiver/audit_logs_archiver.ts

```diff
@@ -0,0 +1,188 @@
+import { Handler } from "aws-lambda";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { BatchWriteCommand, QueryCommand } from "@aws-sdk/lib-dynamodb";
+import { S3Client, PutObjectCommand } from "@aws-sdk/client-s3";
+
+const REGION = process.env.REGION;
+const LOCALSTACK = process.env.LOCALSTACK;
+const AUDIT_LOGS_DYNAMODB_TABLE_NAME = process.env.AUDIT_LOGS_DYNAMODB_TABLE_NAME;
+const AUDIT_LOGS_ARCHIVE_STORAGE_S3_BUCKET = process.env.AUDIT_LOGS_ARCHIVE_STORAGE_S3_BUCKET;
+const PROCESSING_CHUNK_SIZE = 100;
+
+interface AuditLog {
+  "UserID": string;
+  "TimeStamp": number;
+  "Event#SubjectID#TimeStamp": string;
+  [key: string]: any;
+}
+
+export const handler: Handler = async () => {
+  try {
+    const dynamodbClient = new DynamoDBClient({
+      region: REGION ?? "ca-central-1",
+      ...(LOCALSTACK === "true" && {
+        endpoint: "http://host.docker.internal:4566",
+      }),
+    });
+
+    const s3Client = new S3Client({
+      region: REGION ?? "ca-central-1",
+      ...(LOCALSTACK === "true" && {
+        endpoint: "http://host.docker.internal:4566",
+        forcePathStyle: true,
+      }),
+    });
+
+    await archiveAuditLogs(dynamodbClient, s3Client);
+  } catch (error) {
+    // Error message will be sent to slack
```

Comment:
you need a subscription filter

Reply:
Good catch! Going to prepare a pull request to add the subscription filter